test/grpc/authentication_test.dart

```diff
@@ -44,5 +44,16 @@ void main() {
 
       expect(account, isNotNull);
     });
+
+    test('test session refresh', () async {
+      final session = await client.authenticateDevice(
+        deviceId: faker.guid.guid(),
+      );
+
+      final newSession = await client.sessionRefresh(session: session);
+
+      expect(newSession.token, isNot(equals(session.token)));
+      expect(newSession.expiresAt, greaterThan(session.expiresAt));
```

Comment:
This is test is failing because the server is responding with the same token.
I've changed the `session.token_expiry_sec` a bit but it doesn't help. Any ideas?